sql_generators/active_users_aggregates_v3/templates/desktop_checks.sql

```diff
@@ -165,4 +165,29 @@ SELECT
     NULL
   );
 
+WITH dau_current AS (
+  SELECT
+    SUM(dau) AS dau
+  FROM
+    `{{ project_id }}.{{ dataset_id }}.{{ table_name }}`
+  WHERE
+    submission_date = @submission_date
+),
+dau_previous AS (
+  SELECT
+    SUM(dau) AS dau
+  FROM
+    `{{ project_id }}.{{ dataset_id }}.{{ table_name }}`
+  WHERE
+    submission_date = DATE_SUB(@submission_date, INTERVAL 1 DAY)
+)
+SELECT
+  IF(
+    ABS((SELECT SUM(dau) FROM dau_current) / (SELECT SUM(dau) FROM dau_previous)) > 1.5,
```

Comment:
Not sure why you would need the `ABS` here? 
But if it's to account for the unlikely case of negative DAU (?) what about securing against dividing by zero?